tests/modeladmin/tests.py

```diff
@@ -81,6 +85,13 @@ def get_fieldsets(self, request, obj=None):
         form = ma.get_formset(None).form
         self.assertEqual(form._meta.fields, ['day', 'transport'])
 
+    def test_get_fields(self):
```

Comment:
I've proposed https://github.com/django/django/pull/8591 to eliminate the need for this test.